tests/functional/test_credentials.py

```diff
@@ -1060,7 +1060,7 @@ def test_token_chosen_from_provider(self):
         session = Session(profile='sso-test')
         with SessionHTTPStubber(session) as stubber:
             self.add_credential_response(stubber)
-            stubber.add_response()
+            stubber.add_response(body=b'<Test></Test>')
```

Comment:
Yes, this change was introduced because the current tests fail as they go through the `check_for_200_error` handler, which expects certain operations to include a body.

Reply:
Adding bodies to our test suite is breaking changes. This will be reverted in the next revision.